config/development/kustomization.yaml

```diff
@@ -35,6 +35,6 @@ resources:
 - crd
 - ../rbac
 - ../manager
-- ../webhook
+- webhook
```

Comment:
Instead of duplicating the entire webhook folder in the development variant, maybe we can still reference the default `../webhook` variant and only add the mutating webhooks for v1beta1 CRDs in the webhook folder for development variant?

Reply:
duplicates removed, the mutatingwebhook resource now patched for dev version